server/views/pages/support/offenderDetail.njk

```diff
@@ -92,7 +92,7 @@
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             <table class="govuk-table">
-                <caption class="govuk-table__caption govuk-table__caption--m">Sentence details</caption>
+                <caption class="govuk-table__caption govuk-table__caption--m">Sentence details<p>This information comes from NOMIS.</caption>
```

Comment:
I don't think the `<p>` is meant to be here? If it is, it should be outside of the <caption></caption> and should have a closing tag.